botfront/imports/api/graphql/botResponses/resolvers/nlgResolver.js

```diff
@@ -24,7 +16,7 @@ const chooseTemplateSource = (responses, channel) => {
 };
 
 const resolveTemplate = async ({
-    template, projectId, language, slots, channel = null,
```

Comment:
I see slots being remove from the parameters but I don't see the call to that function being changed to reflect that